packages/common/src/services/wallet-client/WalletClient.ts

```diff
@@ -32,14 +33,22 @@ export class WalletClient {
   }
 
   /** Get user's current ETH Audio balance. Returns null on failure. */
-  async getCurrentBalance(_bustCache = false): Promise<BNWei | null> {
-    return null
+  async getCurrentBalance(bustCache = false): Promise<BNWei | null> {
```

Comment:
Very scary - this got cannibalized in #6185 likely due to a typo + save-with-autofix combo 😭 (the autofix deletes dead code, which is sometimes unintentionally created with misplaced parens/brackets). I combed through the PR again to make sure it didn't happen anywhere else.